bolt/test/X86/Inputs/helperSafeICFICPTest.s

```diff
@@ -0,0 +1,40 @@
+# clang++ -O2 helper.cpp -c -o helperProf.o
+# int returnFive() {
+#   return 5;
+# }
+# int returnFourOrFive(int val) {
+#   return val == 1 ? 4 : 5;
+# }
+
+	.text
+	.file	"helper.cpp"
+	.globl	_Z10returnFivev                 # -- Begin function _Z10returnFivev
```

Comment:
Please remove comments and use simple non-mangled names.

Reply:
Other tests like infer_no_exit.s or cmove-conversion.s has them.
This seems like busy work.